### PR TITLE
DOC-102: Overlays JSDoc Update

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namepaces:
- Overlays

Also:
- Some improvements and fixes to the Entities API JSDoc per Overlays API JSDoc.
- Miscellaneous minor fixes to other JSDoc noticed in passing.

Primary case: https://highfidelity.atlassian.net/browse/DOC-102

Other cases:
- https://highfidelity.atlassian.net/browse/DOC-65

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20322/HighFidelity-Beta-PR15913-2cb3425.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20322/HighFidelity-Beta-Interface-PR15913-2cb3425.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20322/HighFidelity-Beta-PR15913-2cb3425.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20322/HighFidelity-Beta-Interface-PR15913-2cb3425.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20322/HighFidelity-Beta-PR15913-2cb3425.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20322/HighFidelity-Quest-Beta-PR15913-2cb3425.apk)

File: interface/src/ui/overlays/Overlays.cpp [hunk 1]
```diff
@@ -1333,127 +1333,170 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ *     <tr><td><code>"circle3d"</code></td><td>3D</td>
+ *       <td>A circle.<br/>
+ *       <span class="important">Deprecated.</span></td>
+ *       <td>{@link Overlays.OverlayProperties-Circle3D|OverlayProperties-Circle3D}</td></tr>
  *   </tbody>
  * </table>
  * <p>2D overlays are rendered on the display surface in desktop mode and on the HUD surface in HMD mode. 3D overlays are
- * rendered at a position and orientation in-world, but are deprecated (use local entities instead).<p>
- * <p>Each overlay type has different {@link Overlays.OverlayProperties|OverlayProperties}.</p>
+ * rendered at a position and orientation in-world.<p>
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 * rendered at a position and orientation in-world.</p>
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 2]
```diff
@@ -1333,127 +1333,170 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ * <p class="important">3D overlays are deprecated: Use local {@link Entities} for these instead.</p>
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 * <p class="important">3D overlays are deprecated. Use local {@link Entities} instead.</p>
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 3]
```diff
@@ -1514,24 +1566,34 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ *     the HUD surface.
+ * @property {boolean} drawHUDLayer=false - <code>true</code> if the overlay is rendered in front of everything, including the
+ *     HUD surface.
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
- *
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 4]
```diff
@@ -1553,20 +1615,27 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ *     the HUD surface.
+ * @property {boolean} drawHUDLayer=false - <code>true</code> if the overlay is rendered in front of everything, including the
+ *     HUD surface.
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
- *
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is 
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather 
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 5]
```diff
@@ -1619,21 +1690,31 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ * @property {boolean} drawInFront=false - <code>true</code> if the overlay is rendered on top of the world layer but behind
+ *     the HUD surface.
+ * @property {boolean} drawHUDLayer=false - <code>true</code> if the overlay is rendered in front of everything, including the
+ *     HUD surface.
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 6]
```diff
@@ -1646,42 +1727,39 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 7]
```diff
@@ -1703,33 +1781,29 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
- * @property {number} textAlpha=1 - The text alpha value.
- * @property {Color} backgroundColor=0,0,0 - The background color.
- * @property {number} backgroundAlpha=0.7 - The background alpha value.
- * @property {number} lineHeight=1 - The height of a line of text in meters.
- * @property {number} leftMargin=0.1 - The left margin, in meters.
- * @property {number} topMargin=0.1 - The top margin, in meters.
- * @property {number} rightMargin=0.1 - The right margin, in meters.
- * @property {number} bottomMargin=0.1 - The bottom margin, in meters.
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 8]
```diff
@@ -1751,29 +1825,31 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ * @property {boolean} drawInFront=false - <code>true</code> if the overlay is rendered on top of the world layer but behind
+ *     the HUD surface.
+ * @property {boolean} drawHUDLayer=false - <code>true</code> if the overlay is rendered in front of everything, including the
+ *     HUD surface.
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 9]
```diff
@@ -1795,29 +1871,31 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 10]
```diff
@@ -1827,35 +1905,61 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 11]
```diff
@@ -1877,25 +1981,28 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
- *
- * @property {boolean} followCamera=true - If <code>true</code>, the grid is always visible even as the camera moves to another position.
- * @property {number} majorGridEvery=5 - Integer number of <code>minorGridEvery</code> intervals at which to draw a thick grid line. Minimum value = <code>1</code>.
- * @property {number} minorGridEvery=1 - Real number of meters at which to draw thin grid lines. Minimum value = <code>0.001</code>.
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.cpp [hunk 12]
```diff
@@ -1912,62 +2019,75 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
+ * @property {boolean} drawInFront=false - <code>true</code> if the overlay is rendered on top of the world layer but behind
+ *     the HUD surface.
+ * @property {boolean} drawHUDLayer=false - <code>true</code> if the overlay is rendered in front of everything, including the
+ *     HUD surface.
+ * @property {boolean} grabbable=false - <code>true</code> if the overlay can be grabbed, <code>false</code> if it can't be.
  * @property {Uuid} parentID=null - The avatar, entity, or overlay that the overlay is parented to.
- * @property {number} parentJointIndex=65535 - Integer value specifying the skeleton joint that the overlay is attached to if
- *     <code>parentID</code> is an avatar skeleton. A value of <code>65535</code> means "no joint".
+ * @property {number} parentJointIndex=65535 - Integer value specifying the joint of the entity or avatar that the entity is
+ *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parents's position and orientation rather
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.cpp]:
```suggestion
 *     parented to if <code>parentID</code> is set. Use 65535 or -1 to parent to the parent's position and orientation rather
```

File: interface/src/ui/overlays/Overlays.h [hunk 13]
```diff
@@ -163,12 +171,11 @@ public slots:
     bool editOverlay(const QUuid& id, const QVariant& properties);
 
     /**jsdoc
-     * Edit multiple overlays' properties.
+     * Edits multiple overlays' (or entities') properties.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     * Edits the properties of multiple overlays (or entities).
```

File: interface/src/ui/overlays/Overlays.h [hunk 14]
```diff
@@ -279,10 +312,11 @@ public slots:
     QUuid getOverlayAtPoint(const glm::vec2& point);
 
     /**jsdoc
-     * Get the value of a 3D overlay's property.
+     * Gets the value of a 3D overlay's (or entity's) property.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     * Gets a specified property value of a 3D overlay (or entity).
```

File: interface/src/ui/overlays/Overlays.h [hunk 15]
```diff
@@ -298,12 +332,13 @@ public slots:
     QVariant getProperty(const QUuid& id, const QString& property);
 
     /**jsdoc
-     * Get the values of an overlay's properties.
+     * Gets the values of a 3D overlay's (or entity's) properties.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]: See above.

File: interface/src/ui/overlays/Overlays.h [hunk 16]
```diff
@@ -317,11 +352,11 @@ public slots:
     QVariantMap getProperties(const QUuid& id, const QStringList& properties);
 
     /**jsdoc
-     * Get the values of multiple overlays' properties.
+     * Gets the values of multiple overlays' (or entities') properties.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]: See above.

File: interface/src/ui/overlays/Overlays.h [hunk 17]
```diff
@@ -345,22 +380,20 @@ public slots:
...
-     * @param {Array.<Uuid>} [include=[]] - If not empty then the search is restricted to these overlays.
-     * @param {Array.<Uuid>} [discard=[]] - Overlays to ignore during the search.
-     * @param {boolean} [visibleOnly=false] - If <code>true</code> then only entities that are 
-     *     <code>{@link Entities.EntityProperties|visible}<code> are searched.
-     * @param {boolean} [collideableOnly=false] - If <code>true</code> then only entities that are not 
-     *     <code>{@link Entities.EntityProperties|collisionless}</code> are searched.
-     * @returns {Overlays.RayToOverlayIntersectionResult} The closest 3D overlay intersected by <code>pickRay</code>, taking
-     *     into account <code>overlayIDsToInclude</code> and <code>overlayIDsToExclude</code> if they're not empty.
+     * @param {boolean} [precisionPicking=false] - <code>true</code> to pick against precise meshes, <code>false</code> to pick 
+     *     against coarse meshes. If <coide>true</code> and the intersected entity is a model, the result's
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     *     against coarse meshes. If <code>true</code> and the intersected entity is a model, the result's 
```

File: interface/src/ui/overlays/Overlays.h [hunk 18]
```diff
@@ -345,22 +380,20 @@ public slots:
...
-     * @param {boolean} [visibleOnly=false] - If <code>true</code> then only entities that are 
-     *     <code>{@link Entities.EntityProperties|visible}<code> are searched.
-     * @param {boolean} [collideableOnly=false] - If <code>true</code> then only entities that are not 
-     *     <code>{@link Entities.EntityProperties|collisionless}</code> are searched.
-     * @returns {Overlays.RayToOverlayIntersectionResult} The closest 3D overlay intersected by <code>pickRay</code>, taking
-     *     into account <code>overlayIDsToInclude</code> and <code>overlayIDsToExclude</code> if they're not empty.
+     * @param {boolean} [precisionPicking=false] - <code>true</code> to pick against precise meshes, <code>false</code> to pick 
+     *     against coarse meshes. If <coide>true</code> and the intersected entity is a model, the result's 
+     *     <code>extraInfo</code> property includes more information than it otherwise would.
+     * @param {Array.<Uuid>} [include=[]] - If not empty then the search is restricted to these overlays (and local entities).
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     * @param {Array.<Uuid>} [include=[]] - If not empty, then the search is restricted to these overlays (and local entities).
```

File: interface/src/ui/overlays/Overlays.h [hunk 19]
```diff
@@ -429,63 +465,68 @@ public slots:
...
+     * Script.setTimeout(function() {
+     *     var textSize = Overlays.textSize(overlay, "hello");
+     *     print("Size of \"hello\": " + JSON.stringify(textSize));
+     * }, 500);
      */
     QSizeF textSize(const QUuid& id, const QString& text);
 
     /**jsdoc
-     * Get the width of the window or HUD.
+     * Gets the width of the Interface window or HUD.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     * Gets the width of the Interface window or HUD surface.
```

File: interface/src/ui/overlays/Overlays.h [hunk 20]
```diff
@@ -429,63 +465,68 @@ public slots:
...
      * @function Overlays.width
-     * @returns {number} The width, in pixels, of the Interface window if in desktop mode or the HUD if in HMD mode.
+     * @returns {number} The width, in pixels, of the Interface window if in desktop mode or the HUD surface if in HMD mode.
      */
     float width();
 
     /**jsdoc
-     * Get the height of the window or HUD.
+     * Gets the height of the Interface window or HUD.
```
Comment (jennaingersoll) [inline interface/src/ui/overlays/Overlays.h]:
```suggestion
     * Gets the height of the Interface window or HUD surface.
```

File: interface/src/scripting/WindowScriptingInterface.h [hunk 21]
```diff
@@ -666,7 +666,7 @@ private slots:
     /**jsdoc
      * Triggered when the interstitial mode changes.
      * @function Window.interstitialModeChanged
-     * @param {bool} interstitialMode - The new interstitial mode value. If <code>true</code>, the interstitial graphics are 
+     * @param {boolean} interstitialMode - The new interstitial mode value. If <code>true</code>, the interstitial graphics are
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply interface/src/scripting/WindowScriptingInterface.h]: Done, plus a similar fix for the preceding signal.

File: interface/src/ui/overlays/Overlays.cpp (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 2 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Done.

File: interface/src/ui/overlays/Overlays.cpp (hunk 3 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 4 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 5 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 6 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 7 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 8 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 9 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 10 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 11 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.cpp (hunk 12 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: interface/src/ui/overlays/Overlays.h (hunk 13 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

File: interface/src/ui/overlays/Overlays.h (hunk 14 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

File: interface/src/ui/overlays/Overlays.h (hunk 16 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

File: interface/src/ui/overlays/Overlays.h (hunk 17 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Fixed.

File: interface/src/ui/overlays/Overlays.h (hunk 18 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

File: interface/src/ui/overlays/Overlays.h (hunk 19 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

File: interface/src/ui/overlays/Overlays.h (hunk 20 above)
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.h]: Done.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20442/HighFidelity-Beta-PR15913-3a3bfd1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20442/HighFidelity-Beta-Interface-PR15913-3a3bfd1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20442/HighFidelity-Beta-PR15913-3a3bfd1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20442/HighFidelity-Beta-Interface-PR15913-3a3bfd1.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20442/HighFidelity-Beta-PR15913-3a3bfd1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20442/HighFidelity-Quest-Beta-PR15913-3a3bfd1.apk)

**Pull request merged**